garmin_gpi.cc

```diff
@@ -792,10 +801,7 @@ wdata_alloc()
 static void
 wdata_free(writer_data_t* data)
```

Comment:
...and this becomes the dtor for writer_data about that same day. (I understand gravity and respect not getting sucked into it...)